Examples/Tests/circuits/inputs_longdomain

```diff
@@ -0,0 +1,62 @@
+################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 100000
+amr.n_cell = 8 16000 8
+amr.max_grid_size = 16000
+amr.blocking_factor = 8
+geometry.coord_sys   = 0
+geometry.prob_lo     =  -4.e-6 -0.8e-2 -4.e-6
+geometry.prob_hi     =   4.e-6  0.8e-2  4.e-6
+amr.max_level = 0
+# use pec instead of pml overlaying current source so you don't get a reflection
+boundary.field_lo = periodic pec periodic
+boundary.field_hi = periodic pml periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.use_filter = 0
+warpx.cfl = 0.9
+
+algo.em_solver_medium = macroscopic # vacuum/macroscopic
+
+algo.macroscopic_sigma_method = laxwendroff # laxwendroff or backwardeuler
```

Comment:
this is very minor -- backwardeuler is more stable than laxwendroff and its the default.

Reply:
sure - sigma is zero so it won't matter either way

Reply:
```suggestion
algo.macroscopic_sigma_method = backwardeuler # laxwendroff or backwardeuler
```